lighthouse-cli/test/fixtures/byte-efficiency/tester.html

```diff
@@ -55,6 +55,10 @@
   <!-- PASS(responsive): image is used at full size -->
   <img src="lighthouse-unoptimized.jpg">
 
+  <!-- FAIL(optimized): image is not optimized -->
+  <!-- PASS(responsive): image is used at full size -->
+  <img src="http://localhost:10503/byte-efficiency/lighthouse-unoptimized.jpg">
```

Comment:
and reused the image, nice :) If we had better smokehouse comparators would you also be able to assert that this goes through your new cross origin path?